src/components/charts/galaxySelector/index.jsx

```diff
@@ -317,6 +318,24 @@ class GalaxySelector extends React.PureComponent {
     }));
   };
 
+  getAlertDaysAndHours = id => {
+    const { activeGalaxy } = this.props;
+    const { alerts } = activeGalaxy || {};
```

Comment:
Aren't `alerts` already delivered as a prop?

Reply:
Yes and are exactly the same alerts I'm grabbing out of the `activeGalaxy` prop.